hercules-ci-api/src/Hercules/API/BillingStatus.hs

```diff
@@ -0,0 +1,23 @@
+{-# LANGUAGE DeriveAnyClass #-}
+
+module Hercules.API.BillingStatus
+  ( BillingStatus (..),
+    fromText
+    )
+where
+
+import Hercules.API.Prelude
+
+data BillingStatus
+  = Community -- Free plan
+  | Trial
+  | Active
+  | Cancelled
```

Comment:
Isn't `Cancelled` the same as community?

Reply:
Partially, it allows for frontend to warn the user that the plan has been cancelled.